packages/menu-bar/test/menu-bar.test.js

```diff
@@ -722,6 +723,21 @@ describe('item components', () => {
     expect(item.getAttribute('role')).to.not.equal('menuitem');
   });
 
+  it('should restore menu bar item attribute state when moved from sub-menu back to menu bar', async () => {
+    const item = buttons[5].firstChild;
+    const initialAttributesState = item.getAttributeNames();
```

Comment:
Consider using a shorter variable name e.g. `itemAttributes` sounds reasonable to me.

Reply:
ok, no problem